### PR TITLE
Add improved docs

### PR DESCRIPTION
**Pull request by wooorm**
### Initial checklist

*   [x] I read the support docs 
*   [x] I read the contributing guide 
*   [x] I agree to follow the code of conduct 
*   [x] I searched issues and couldn’t find anything (or linked relevant results below) 
*   [x] If applicable, I’ve added docs and tests

### Description of changes

[View rendered](https://github.com/rehypejs/rehype-meta/tree/docs)

Related-to: unifiedjs/unified#168.

/cc @nemo-omen @remcohaszing @ChristianMurphy @Murderlon @with-heart @joostdecock @mgan59

### CODE REVIEW

**Review by wooorm — COMMENTED** [review]
✅

File: readme.md [hunk 1]
```diff
@@ -8,101 +8,145 @@
...
 *   [Security](#security)
 *   [Related](#related)
 *   [Contribute](#contribute)
 *   [License](#license)
 
-## Install
+## What is this?
+
+This package is a [unified][] ([rehype][]) plugin to manage the metadata (Open
+Graph, Twitter Cards, other SEO things) that can be found in `<head>`.
```
Comment (ChristianMurphy) [inline readme.md]: This feels a bit off, meta is more than SEO.
Maybe reframing this as "this plugin adds SEO metadata, using the meta tag in the head of the document"?

Comment (wooorm) [reply readme.md]: it does do more than just SEO. But it doesn’t do all metadata.

What if the word `other` is removed?

**Review by Murderlon — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (ChristianMurphy) [reply readme.md]: Maybe, or adding an item to the list which is not SEO related

**Pull request merged**